conan/tools/cmake/cmakedeps/templates/target_configuration.py

```diff
@@ -121,6 +109,9 @@ def template(self):
                      $<$<CONFIG:{{configuration}}>:${{'{'}}{{pkg_name}}_LIB_DIRS{{config_suffix}}}> APPEND)
         {%- endif %}
 
+        # For the modules (FindXXX)
+        list(APPEND {{ pkg_name }}_LIBRARIES{{ config_suffix }} {{root_target_name}})
```

Comment:
Note, here I'm attaching the targets, not the path to the libraries like in previous versions. That is because the macro `conan_package_library_targets` was simplified and only returns the targets. The question is, it should work exactly the same right?
When someone uses the `Boost_LIBRARIES` is typically to link a target with them so it should be ok and it is wide easy.